tewma/settings.py

```diff
@@ -84,6 +84,8 @@
     }
 }
 
+MEDIA_URL = '/media/'
```

Comment:
both lines are unnecessary because we have both line in 128-130 https://github.com/Bouncyyahomie/TEWMA-project/blob/cde98dcc6a578ca1ec36a6e836901de87f37b27c/tewma/settings.py#L128-L130